### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by aryancholakkal**
Created branch develop, feature
Added features
Final PR

### CODE REVIEW

File: Aryan_C_book_app/add_book_feature/list_all_books.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+List all books will list all books in ascending order.
```
Comment (elementaryrock) [inline Aryan_C_book_app/add_book_feature/list_all_books.txt:1]: There should be two sentences here

**Review by elementaryrock — CHANGES_REQUESTED** [review]

**Comment by aryancholakkal** [comment]
Done the changes requested

File: Aryan_C_book_app/add_book_feature/list_all_books.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
...
```
Comment (elementaryrock) [inline Aryan_C_book_app/add_book_feature/list_all_books.txt:1]: Pls add in "This is the feature which shows the user all the books they've created as a list."

File: Aryan_C_book_app/add_book_feature/list_all_books.txt [hunk 3]
```diff
@@ -0,0 +1,3 @@
...
+This is a new feature.
```
Comment (elementaryrock) [inline Aryan_C_book_app/add_book_feature/list_all_books.txt:2]: Remove line 2 and 3

**Review by elementaryrock — CHANGES_REQUESTED** [review]

**Pull request merged**